src/lib/forms/rjsf/widgets/RJSFReferencedProvider.js

```diff
@@ -1,12 +1,12 @@
-import { libraryApi } from '@api/ill';
+import { providerApi } from '@api/providers';
```

Comment:
I think that after changing this one and deleting the `ReferencedAcqVendor`, you are missing to edit the `customWidgets` object in `RJSForm.js`, otherwise will not work.
Can you try to create/edit an order and see if the widget to select the provider works?

Reply:
I modified the `customWidgets` [here](https://github.com/KonstantinaStoikou/react-invenio-app-ils/blob/18e97e6d4e7322a559da0ee6b0bbf06b26449ec4/src/lib/forms/rjsf/RJSForm.js#L33)
The widget in orders works correctly